pkg/k8shandler/elasticsearch.go

```diff
@@ -670,13 +698,82 @@ func curlESService(clusterName, namespace string, payload *esCurlStruct, client
 		return
 	}
 
+	request.Header = ensureTokenHeader(request.Header)
 	httpClient := getClient(clusterName, namespace, client)
 	resp, err := httpClient.Do(request)
 
+	if resp != nil {
+		// TODO: eventually remove after all ES images have been updated to use SA token auth for EO?
+		if resp.StatusCode == http.StatusForbidden {
```

Comment:
There is also the case that the token was not able to be read, and so there is no token in the header.  Not sure if you will get a `401` or `403` in that case - probably the latter.

Reply:
`http.StatusForbidden` is a `403`, i'll test to see what an empty token would return us to see if we need to expand our status code check